src/Types.tsx

```diff
@@ -0,0 +1,39 @@
+export type ResponseSectionsResults = {
+    apiUrl: string;
+    id: string;
+    webTitle: string;
+    webUrl: string;
+}
+
+export type ResponseSearchResults = {
+    id: string;
+    type: string;
+    sectionId: string;
+    sectionName: string;
+    webPublicationDate: string;
+    webTitle: string;
+    webUrl: string;
+    apiUrl: string;
+    isHosted: boolean;
+    pillarId: string;
+    pillarName: string;
+}
+
+export type ResponseSearch = {
+    status: string;
+    userTier: string;
+    total: number;
+    startIndex: number;
+    pageSize: number;
+    currentPage: number;
+    pages: number;
+    orderBy: string;
+    results: ResponseSearchResults[];
+}
+
+export interface Author {
```

Comment:
Is there any specific reason why you've used here `interface` for defining type and above you've used `type` for defining types?